package.json

```diff
@@ -28,7 +28,7 @@
     "@material-ui/core": "^4.11.4",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.58",
-    "@olympusdao/component-library": "^1.1.0",
+    "@olympusdao/component-library": "^1.2.2",
```

Comment:
Could you rev this to 1.3.0? There's another PR out there with this higher version. Don't want to unintentionally downgrade.